src/merkleblock.h

```diff
@@ -157,17 +161,10 @@ class CMerkleBlock
 
     CMerkleBlock() {}
 
-    ADD_SERIALIZE_METHODS;
-
-    template <typename Stream, typename Operation>
-    inline void SerializationOp(Stream& s, Operation ser_action) {
-        READWRITE(header);
-        READWRITE(txn);
-    }
-
-private:
```

Comment:
In commit "Convert merkleblock to new serialization" (25bfbbeb63eca1f1c8852a8a08844069d9e06776)

Maybe better to keep constructor below private and just declare SERIALIZE_METHODS before it, assuming there's a valid reason for the constructor to be private.

Reply:
I think this was a mistake; fixed.